lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -140,6 +282,128 @@ class RequestManager {
   }
 }
 
+class PerDocumentResourceLoader {
+  constructor(document) {
+    this._document = document;
+    this._defaultEncoding = document._encoding;
+    this._resourceLoader = document._defaultView ? document._defaultView._resourceLoader : new ResourceLoader();
```

Comment:
Hmm, for documents without a _defaultView, we should generally not be loading resources, I am pretty sure. Does this happen in the tests or something?

Reply:
I think I did this before testing, I can try to remove the default resource loader.